specs/packages/sdk.md

```diff
@@ -44,3 +44,36 @@ Sdk struct wraps logic from the other modules, creating public API using `wasm_b
 
 At the moment, the client of the Sdk is responsible for keeping storage in sync with the Sdk's wallet instance. What does that mean? The Namada extension has its own key storage, which is modified by user actions. Every time the extension storage changes (for example, when a new address is derived), we must manually sync the extension and Sdk wallet storage
 using functions provided by `wallet` module. This is something that will most likely change in the future.
+
+## How can you import and use the SDK?
+First you need to build the shared package. We do that using `wasm-pack`.
+You can take a look at [build.sh](https://github.com/anoma/namada-interface/blob/main/packages/shared/scripts/build.sh) and wasm-pack specific configuration in [Cargo.toml](https://github.com/anoma/namada-interface/blob/main/packages/shared/lib/Cargo.toml)
+After build is done you should see built files in the `src/shared/*`
+```
+LICENSE_MIT
+package.json
+README.md
+shared.d.ts
+shared.js
+shared_bg.js
+shared_bg.wasm
+shared_bg.wasm.d.ts
+```
+You can take a look at `shared.d.ts` to see what is exported.
+
+Next step would be to import SDK itself.
+For now we wrap all of the SDK logic in the [Sdk struct](https://github.com/anoma/namada-interface/blob/main/packages/shared/lib/src/sdk/mod.rs) and [Query struct](https://github.com/anoma/namada-interface/blob/main/packages/shared/lib/src/query.rs). Main reason we do that is to not pass things like `HttpClient`, `Wallet` and `ShieldedContext` every time we want to submit new transaction or query data. This will most likely change in future as we are sometimes getting "multiple mutable references" Rust error.
+
+To use structs (almost)all you have to do is to create new instance:
```

Comment:
> To use structs (almost)all you have to do is to create new instance:

There's a missing space, but I also think we could reword this to get rid of it entirely:

> Within the extension, we create instances of `Sdk` and `Query` as follows: